### ISSUE TITLE
Error in building 

### ISSUE THREAD

**Issue by jimchen1980** [peterwilli/IOTA-tangle-explorer#4]
Chunk Names
               static/js/vendor.5e5a0381ee3bb07fbe8d.js    3.62 MB       0  [emi                                   tted]  [big]  vendor
                  static/js/app.2d4c5402137aad281620.js     224 kB       1  [emi                                   tted]         app
             static/js/manifest.a74e134a1093d40250ce.js    1.51 kB       2  [emi                                   tted]         manifest
    static/css/app.0d2897a5be64bc6b24e7e7e7679e3cf0.css    9.69 kB       1  [emi                                   tted]         app
           static/js/vendor.5e5a0381ee3bb07fbe8d.js.map    4.46 MB       0  [emi                                   tted]         vendor
              static/js/app.2d4c5402137aad281620.js.map    1.74 MB       1  [emi                                   tted]         app
static/css/app.0d2897a5be64bc6b24e7e7e7679e3cf0.css.map    21.6 kB       1  [emi                                   tted]         app
         static/js/manifest.a74e134a1093d40250ce.js.map    14.6 kB       2  [emi                                   tted]         manifest
                                             index.html  522 bytes          [emi                                   tted]

ERROR in static/js/vendor.5e5a0381ee3bb07fbe8d.js from UglifyJs
Unexpected token: name (Ceri) [./~/ceri/lib/wrapper.js:15,0][static/js/vendor.5e                                   5a0381ee3bb07fbe8d.js:46727,16]

  Build complete.

  Tip: built files are meant to be served over an HTTP server.
  Opening index.html over file:// won't work.

root@iZj6c1hdhubslyveeg4tskZ:/data/wwwroot/iota.oxifus.com# npm run build

> tangle-explorer-web@1.0.0 build /data/wwwroot/iota.oxifus.com
> node build/build.js

⠴ building for production...[BABEL] Note: The code generator has deoptimised the                                    styling of "/data/wwwroot/iota.oxifus.com/src/lib/iota.js" as it exceeds the ma                                   x of "500KB".
⠼ building for production...
Starting to optimize CSS...
Processing static/css/app.0d2897a5be64bc6b24e7e7e7679e3cf0.css...
Processed static/css/app.0d2897a5be64bc6b24e7e7e7679e3cf0.css, before: 10023, after: 9691, ratio: 96.69%
Hash: c03589392573a3263093
Version: webpack 2.7.0
Time: 24785ms
                                                  Asset       Size  Chunks                    Chunk Names
               static/js/vendor.5e5a0381ee3bb07fbe8d.js    3.62 MB       0  [emitted]  [big]  vendor
                  static/js/app.2d4c5402137aad281620.js     224 kB       1  [emitted]         app
             static/js/manifest.a74e134a1093d40250ce.js    1.51 kB       2  [emitted]         manifest
    static/css/app.0d2897a5be64bc6b24e7e7e7679e3cf0.css    9.69 kB       1  [emitted]         app
           static/js/vendor.5e5a0381ee3bb07fbe8d.js.map    4.46 MB       0  [emitted]         vendor
              static/js/app.2d4c5402137aad281620.js.map    1.74 MB       1  [emitted]         app
static/css/app.0d2897a5be64bc6b24e7e7e7679e3cf0.css.map    21.6 kB       1  [emitted]         app
         static/js/manifest.a74e134a1093d40250ce.js.map    14.6 kB       2  [emitted]         manifest
                                             index.html  522 bytes          [emitted]

ERROR in static/js/vendor.5e5a0381ee3bb07fbe8d.js from UglifyJs
Unexpected token: name (Ceri) [./~/ceri/lib/wrapper.js:15,0][static/js/vendor.5e5a0381ee3bb07fbe8d.js:46727,16]

**Comment by peterwilli** [comment]
What OS are you using?

**Comment by jimchen1980** [comment]
Ubuntu 16.04

**Comment by peterwilli** [comment]
I'm using the same. I'm currently helping someone with the same problem (I think). I had the same problem too but forgot to document how I solved it. So I'm currently spawning a ubuntu VM and try to reproduce

**Comment by peterwilli** [comment]
@jimchen1980 btw does it say build complete on the bottom? I have the same error but the build runs fine. Also if you want to test you can run `npm run dev` :) It'll open a browser with local page.
![selection_001](https://user-images.githubusercontent.com/1212814/29310922-fc63bae8-81af-11e7-9aa8-776e964ab952.png)

**Comment by arancauchi** [comment]
This looks like there is a problem translating the ceri-icon dependency during the uglify phase, which shouldn't be a problem after webpack has bundled all dependencies together

**Comment by jimchen1980** [comment]
no, it does not run correctly, and all i only see is a blank page

**Comment by peterwilli** [comment]
I'll try to upgrade ceri-icon. When I started using it, it was in beta, maybe it's upgraded now.
